gulpfile.js

```diff
@@ -0,0 +1,49 @@
+var gulp = require('gulp');
```

Comment:
It's good practice to order this. Something like gulp, gulp-* (a to z), others (a to z)